Source/Set Production Image Page Count v4.krs

```diff
@@ -0,0 +1,253 @@
+﻿<script>
+	<name>Set Production Image Page Count</name>
+	<description>This Relativity action script populates a whole number field with the production image count for documents in the selected saved search and production set. KCD_1037602. Made 2017 - modified by TSD to ensure 9.4 Production consistency.</description>
+	<category>Case Functionality</category>
+	<Version>4.1</Version>
+	<input>
+		<field id="PageCountField" name="Page Count Field:">
+			<filters>
+				<type>1</type>
+				<category>0</category>
+			</filters>
+		</field>
+		<search id="searchId" name="Saved Search:" />
+		<sql id="productionSet" name="Production Set:">
+			SELECT 
+				ID
+				,Display
+			FROM 
+			(
+				SELECT 
+					ID = 0
+					,Display = ' All'
+				UNION ALL
+				SELECT 
+					ID = ArtifactID
+					,Display = Name
+				FROM EDDSDBO.Production WITH(NOLOCK)
+			) AS T
+			ORDER BY Display
+		</sql>
+	</input>
+	<action returns="table" timeout="1200"><![CDATA[
+
+		DECLARE @SQL NVARCHAR(MAX)
+		DECLARE @ParmDefinition NVARCHAR(MAX)
+		DECLARE @error NVARCHAR(MAX)
+		DECLARE @ProductionArtifactID INT = #productionSet#
+		DECLARE @ProductionTableName NVARCHAR(MAX) = '[EDDSDBO].ProductionDocumentFile_' +CAST(@ProductionArtifactID AS NVARCHAR(MAX))
+		DECLARE @newProductionFormatVersion DECIMAL(17,3) = 9.3
+		DECLARE @lookupVersion NVARCHAR(MAX)
+		DECLARE @currentVersion DECIMAL(17,3)
+		DECLARE @productionsAppGuid AS UNIQUEIDENTIFIER = '51B19AB2-3D45-406C-A85E-F98C01B033EC'
+		DECLARE @productionsObjectGuid AS UNIQUEIDENTIFIER = '24190650-2E73-4373-B0C4-F31142CBF300'
+		DECLARE @currentProduction INT
+		
+		IF EXISTS (SELECT * FROM TEMPDB.DBO.SYSOBJECTS O WHERE O.XTYPE IN ('U') AND O.ID = OBJECT_ID(N'TEMPDB..#KCD_00048112_Documents'))
+               DROP TABLE #KCD_00048112_Documents;
+
+        IF EXISTS (SELECT * FROM TEMPDB.DBO.SYSOBJECTS O WHERE O.XTYPE IN ('U') AND O.ID = OBJECT_ID(N'TEMPDB..#KCD_00048112_Productions'))
```

Comment:
Hi! @tsd-mmarinova - I am Priya and reviewing the code change for this change. I am just curious to know if there was any specific reason  why we changed the table from a physical table to a temp table?